scripts/install_azcopy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -x
+pwd
```

Comment:
Do we need to have this script in the public repo?

Could we please move this to `azureml-asset` instead?

Reply:
Was seeing if this will get run as part of assets-test.yml in workflow. But it is getting skipped as label `safe to test` is not set. @elesel @wayliums can you please help to share how to add label `safe to test`. And also, if we can do model assets validation as part of publisher spec PR. In that case we will require `azcopy` installation if model is downloaded and registered to a test workspace.

Reply:
Resolving this. Checking with Chuyuan if we need in this repo